desktop/apps/auction_support/client/bid_form.coffee

```diff
@@ -12,10 +12,12 @@ module.exports = class BidForm extends ErrorHandlingForm
 
   timesPolledForBidPlacement: 0
   maxTimesPolledForBidPlacement: sd.MAX_POLLS_FOR_MAX_BIDS
-  errors:
+  errors: -> {
```

Comment:
In order to pass a function (L17) that can determine the prediction url, I'm making this a function so I can reference `@/this`. See the `error_handling_form` tests below. It's possible there was a better way to do this.